examples/workflow-test/package.json

```diff
@@ -47,8 +47,7 @@
   "devDependencies": {
     "@eclipse-glsp-examples/workflow-server-bundled": "~2.0.1",
     "@eclipse-glsp/glsp-playwright": "~2.0.0",
-    "@playwright/test": "^1.34.3",
-    "@theia/playwright": "^1.39.0",
+    "@playwright/test": "^1.40.1",
     "commander": "^10.0.1",
     "dotenv": "^16.0.3",
     "mvn-artifact-download": "5.1.0",
```

Comment:
Pretty sure we don't need `mvn-artifact-download` as dependency anymore, since we consume the node-server variant from npm.